core/tests/unit/ldml/test_kmx_plus.cpp

```diff
@@ -0,0 +1,10 @@
+#include <stdio.h>
+#include <assert.h>
+#include <kmx/kmx_plus.h>
+
+using namespace km::kbp::kmx;
+
+int main(int argc, const char *argv[]) {
+    printf("OKAY\n");
```

Comment:
This is a stub for now, because I'm doing only static_asserts for now, but may add some more detailed unit tests of kmx_plus functions here.